### DISCUSSION TITLE
Stellarium consistently crashes when changing display device using RDS

### DISCUSSION THREAD

**Discussion by astange2** [Stellarium/stellarium#1974] [General]
Hi,
I have recently started to use Stellarium through Windows Remote Desktop over my local Wi-Fi. Unfortunately using Stellarium in this environment seems unstable as it crashes every time I reconnect the client(which happens quite frequently, either because I change the display device or I need to reconnect my windows client after a lost Wi-Fi connection.) However, this instability only seems to occur when I have connected Stellarium to a telescope through the telescope connection plugin(which I almost always do).
Im running 64-bit windows 10.

Is there a way to solve this issue or is it a bug? Is Windows RDS even
supported by Stellarium or is there another preferred Remote Desktop solution that I should use?

thanks,
Anders

**Comment by gzotti** [comment]
We have seen a few reports about issues around RDS and the Telescope plugin when a telescope is connected on the remote computer. We cannot give guarantees that RDS works, and would like to hear about solutions ourselves. You may want to try another remote desktop program. Please report if that would work. 

Given that Stellarium is a graphics-heavy program, the use of RDS is questionable to begin with. Maybe running it locally and sending only the telescope control commands via network (Indi?) is the technically better solution. Experience reports welcome.

**Comment by astange2** [comment]
﻿Running Stellarium through RDS seems fine in general. I did experience some lagging but after turning down the magnitude limit for stars, it improved.
It’s only in when using an active telescope connection that it actually crashes. 
I will try to explore other RD solutions to see if the issue can be resolved that way.

Thanks for you suggestions. Unfortunately, running Stellarium locally is not an option for me. All my other Astro-software that are required simultaneously, run on my “server” pc which controls all the usb connections(mount, cameras, dome etc).

**Comment by astange2** [comment]
Hi,I just want to report back on the RDS issue I brought up about a year ago(see below). I have tried a few other free Remote Desktop solutions, none good enough in my opinion, until I installed the free version of Microsoft TeamViewer. Stellarium works perfectly in this environment, at least for my purposes, and I have not seen any crashes when switching back and forth between different devices.Best regards,Anders StangePs. Congratulations on version 1.0. Stellarium is really an impressive piece of software.21 okt. 2021 kl. 09:47 skrev Georg Zotti ***@***.***>:﻿
We have seen anfew reports about issues around RDS and the Telescope plugin when a telescope is connected on the remote computer. We cannot give guarantees that RDS works, and would like to hear about solutions ourselves. You may want to try another remote desktop program. Please report if that would work.
Given that Stellarium is a graphics-heavy program, the use of RDS is questionable to begin with. Maybe running it locally and sending only the telescope control commands via network (Indi?) is the technically better solution. Experience reports welcome.
